pkgs/applications/science/logic/beluga/default.nix

```
@@ -1,32 +1,38 @@
-{ stdenv, fetchFromGitHub, ocamlPackages, omake }:
+{ stdenv, fetchFromGitHub, ocamlPackages, rsync }:
 
-stdenv.mkDerivation {
-  name = "beluga-20180403";
+ocamlPackages.buildDunePackage {
+  pname = "beluga";
+  version = "unstable-20200311";
```

```suggestion
  version = "unstable-2020-03-11";
```
Generally we put dashes to make this readable.

---

@bhipple -- sure, updated expression/commit/PR with this convention.